website_and_docs/content/documentation/webdriver/getting_started/install_selenium_library.pt-br.md

```diff
@@ -2,6 +2,7 @@
 title: "Instalando bibliotecas do Selenium"
 linkTitle: "Instalando bibliotecas do Selenium"
 weight: 1
+needsTranslation: true
 description: >
   Setting up the Selenium library for your favourite programming language.
```

Comment:
Configurando a biblioteca Selenium para sua linguagem de programção favorita

Reply:
translation to portuguese from the "Setting up the Selenium library for your favourite programming language."